load-test/query_load.sh

```diff
@@ -6,28 +6,4 @@ then
     ACC=1
 fi
 
-IP="$(dig +short myip.opendns.com @resolver1.opendns.com)"
-echo "Public IP address: ${IP}"
-PORT=16657
-
-if [ -z $IP ]
-then
-    IP=127.0.0.1
-    PORT=16657
-fi
-
-RPC="http://${IP}:${PORT}"
-acc1=$($DAEMON keys show validator$ACC -a --home $DAEMON_HOME-1 --keyring-backend test)
-val1=$($DAEMON keys show validator$ACC -a --bech val --home $DAEMON_HOME-1 --keyring-backend test)
-for (( a=1; a<10000; a++ ))
-do
-	bTx=$("${DAEMON}" q bank balances "${acc1}" --node $RPC --output json)
-	bTxres=$(echo "${bTx}" | jq -r '.balances')
-	echo "** Balance :: $bTxres **"
-	sTx=$("${DAEMON}" q staking validators --node $RPC --output json)
-	sTxres=$(echo "${sTx}" | jq -r '.validators[].description.moniker')
-	echo "** Monikers :: $sTxres **"
-	dTx=$("${DAEMON}" q staking delegation "${acc1}" "${val1}" --node $RPC --output json)
-	dTxres=$(echo "${dTx}" | jq -r '.delegation.shares')
-	echo "** Delegations :: $dTxres **"
-done
+python3 ../python-qa-tools/load-test/query_load.py $ACC
```

Comment:
What is this $ACC here. Please document

Reply:
@juggernaut09 let's use either RANDOM account or chose one of the account. Let's not take user input here. Or else, consider taking address input from user directly. It allows people to perform loadtest on external networks.